docs/content/en/docs/tasks/cluster/manual-renew-certs.md

```diff
@@ -0,0 +1,117 @@
+| peer                  | sa                       |
+| server                | etcd/ca.crt              |
+|                       | apiserver-kubelet-client |
+|                       | apiserver                |
+|                       | front-proxy-client       |
+
+You can renew all certificates in the table except ca by following the steps given below.
+
+### External etcd node
+
+If the cluster has external etcd, you need to renew the certificates in etcd nodes first.
+
+1. ssh into the etcd node and run the following commands:
+
+{{< tabpane >}}
+{{< tab header="Ubuntu or RHEL" lang="bash" >}}
+# backup certs
+cd /etc/etcd
+sudo mv pki pki.bak
+sudo mkdir pki
+sudo mv pki.bak/ca.* pki
+
+# run certificates join phase to regenerate the deleted certificates
+sudo etcdadm join phase certificates http://dumburl
+
+# etcd will auto pick the new certificates
+{{< /tab >}}
+
+{{< tab header="Bottlerocket" lang="bash" >}}
+# get the image name of the kubeadm-bootstrap container
+apiclient get | jq '.settings["host-containers"]["kubeadm-bootstrap"].source'
+
+# go to root shell from control container
+sudo sheltie
+
+# pull the image
+IMAGE_ID=${the image you got from previous step}
+ctr image pull ${IMAGE_ID}
+
+# backup certs
+cd /var/lib/etcd
+mv pki pki.bak
+mkdir pki
+mv pki.bak/ca.* pki
+
+
+# recreate certificates
+ctr run --mount type=bind,src=/var/lib/etcd/pki,dst=/etc/etcd/pki,options=rbind:rw --net-host --rm \
+${IMAGE_ID} tmp-cert-renew \
+/opt/bin/etcdadm join phase certificates http://dumburl --init-system kubelet
+
+# etcd will auto pick the new certificates
+{{< /tab >}}
+{{< /tabpane >}}
+
+2. Verify your etcd node is running correctly
+
+{{< tabpane >}}
+{{< tab header="Ubuntu or RHEL" lang="bash" >}}
+sudo etcdctl --cacert=ca.crt --cert=etcdctl-etcd-client.crt --key=etcdctl-etcd-client.key member list
+{{< /tab >}}
+{{< tab header="Bottlerocket" lang="bash" >}}
+# there should be no error except "connection refused" for the health check
+tail -f -n 100 /var/log/containers/etcd*
+{{< /tab >}}
+{{< /tabpane >}}
+
+3. Repeat the above steps for all etcd nodes.
+
+3. Save api-server-etcd-client crt and key file as secret from one of the etcd nodes, so the key can be picked up by new control plane node. You will also need it when renewing certs in CP node.
+```
+kubectl edit secret ${cluster-name}-api-server-etcd-client -n eksa-system
+```
+
+
+### CP node
+
+1. ssh into the control plane node and run the following commands:
+
+```bash
+sudo kubeadm certs renew all
+```
+
+2. Verify your certs have been renewed.
+
+```bash
+sudo kubeadm certs check-expiration
+```
+
+3. If you have external etcd node, manually replace the api-server-etcd-client.crt and api-server-etcd-client.key file in /etc/kubernetes/pki folder with the one you saved from any etcd node.
+
+4. Restart static control plane pods.
+
+   a. Temporarily move all manifest files from /etc/kubernetes/manifests/ and wait for 20 second.
+
+   b. Move all manifest back.
```

Comment:
In restarting static pods on CP, Ive seen places where 20 seconds wasnt enough for kubelet to stop those pods. Can we add a `crictl ps` to verify `api-server, scheduler, controller-manager` pods not running, and then move manifests back, and once again verify they are running with `crictl ps`?

Reply:
added: `You can verify pods restarting by running kubectl in your admin machine.`